src/renderer/containers/ReplayBrowser/ReplayFile.tsx

```diff
@@ -153,12 +156,27 @@ export const ReplayFile: React.FC<ReplayFileProps> = ({
               >
                 <InfoItem label={<EventIcon />}>{monthDayHourFormat(moment(date))}</InfoItem>
 
-                {lastFrame !== null && (
-                  <InfoItem label={<TimerOutlinedIcon />}>
-                    {convertFrameCountToDurationString(lastFrame, "m[m] ss[s]")}
+                {lastFrame !== null && gameMode !== GameMode.HOME_RUN_CONTEST && (
+                  <InfoItem label={<TimerIcon />}>
+                    {gameMode == GameMode.TARGET_TEST
+                      ? frameToGameTimer(lastFrame, settings)
+                      : convertFrameCountToDurationString(lastFrame, "m[m] ss[s]")}
                   </InfoItem>
                 )}
-                <InfoItem label={<LandscapeIcon />}>{stageName}</InfoItem>
+
+                <InfoItem
+                  label={
+                    gameMode === GameMode.TARGET_TEST ? (
```

Comment:
pick the icon to use right after you get the gameMode from the settings. Use a switch statement for readability.